BUILD.md

```diff
@@ -183,6 +183,10 @@ See more information on the TensorRT Execution Provider [here](./docs/execution_
 
 Dockerfile instructions are available [here](./dockerfiles#tensorrt)
 
+#### Jetson (ARM64 Builds)
+
+See [instructions](https://github.com/microsoft/onnxruntime/issues/2684#issuecomment-568548387) for additional information and tips related to building Onnxruntime with TensorRT Execution Provider on Jetson platforms (TX1/TX2, Nano)
```

Comment:
Will it be good to add CI build for Jetson/ARM?

Reply:
there's work in progress to enable general ARM/IOT device support.